metrics/metrics.go

```diff
@@ -155,17 +174,18 @@ func (s *Store) WriteMetrics(w io.Writer) {
 		}
 
 		avg := time.Duration(int(sum) / len(v))
+		cacheCount := s.cached[k]
 
-		results[i] = result{key: k, count: len(v), max: max, sum: sum, avg: avg, cacheFactor: cacheFactor}
+		results[i] = result{key: k, count: len(v), max: max, sum: sum, avg: avg, cacheCount: cacheCount, cacheFactor: cacheFactor}
 		i++
 	}
 
 	s.mu.Unlock()
 
 	if s.calculateHints {
-		fmt.Fprintf(w, "  %9s  %13s  %12s  %12s  %5s  %s\n", "cache", "cumulative", "average", "maximum", "", "")
-		fmt.Fprintf(w, "  %9s  %13s  %12s  %12s  %5s  %s\n", "potential", "duration", "duration", "duration", "count", "template")
-		fmt.Fprintf(w, "  %9s  %13s  %12s  %12s  %5s  %s\n", "-----", "----------", "--------", "--------", "-----", "--------")
+		fmt.Fprintf(w, "  %9s  %13s  %12s  %12s  %5s  %6s  %s\n", "cache", "cumulative", "average", "maximum", "total", "cached", "")
```

Comment:
OK, taking a second look, I think I have misunderstood the initial implementation. 

We have 

1. Include
2. IncludeCached

Today, we only track the `Include` variant, which, esp. when we add the cached column, does not make sense (as it does not add up to 100%).

I suggest we fix this by:

1. Removing the new method you have now added
2. Add a new attribute (cached) to `TrackValue` and invoke that from both of the methods above.
3.

Reply:
We track both today since `IncludeCached` calls `Include` via [`ns.getOrCreate`](https://github.com/gohugoio/hugo/blob/v0.82.0/tpl/partials/partials.go#L223).  I'll see if there's another way to track these cached calls.

Reply:
That's not what I see in the code, but I could be tired.

Reply:
I've refactored this PR.  Let me know what you think.  See the sample output in the commit message.